ropetest/codeanalyzetest.py

```diff
@@ -432,7 +432,7 @@ def test_renaming_functions_with_from_import_and_parens(self):
         found_pyname = name_finder.get_pyname_at(code.index('afunc') + 1)
         self.assertEquals(afunc.get_object(), found_pyname.get_object())
 
-    @testutils.run_only_for_25
```

Comment:
> (It may no longer be the case due to my py3 PR, I think the new exception syntax was backported as far back as py26 only... I did not realise there was 2.5- support. :s )

Right. So we lost < 2.6 compatibility. Not a big deal. I don't have to look for that VM ;).